test/integration/roles/test_ec2_eip/tasks/main.yml

```
@@ -152,7 +152,7 @@
 - name: assert new EIP was assigned
   assert:
     that:
-      - '"failed" not in result'
```

this should actually be `not result|failed` or `not result.failed`, idem for most of the following

---

Well, the task didn't fail, otherwise we wouldn't be here. We are in fact checking if the key exists (as it should always be). The only reason I added it, is because all the other tasks had it, so I assume this was an oversight.

Checking if the task did not fail is of no use IMO.

---

did not check, normally is done when prev task has `ignore_errors` ... i would just remove then.

---

Ok, but a lot of tasks simply checked that 'failed' was not returned, which we now have modified.
I will remove all of them ?

---

as long as the previous one is not ignoring errors .. this is a useless check

---

Done.